PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Description
+Describe your changes in detail.
+
+## Motivation and Context
+Why is this change required? What problem does it solve? 
+If it fixes an open issue, please link to the issue here. 
+
+## Changes
+What kind of change does this PR introduce? (Bug fix, feature, documentation update...)
+
+## Behavior
+What is the current behavior? What is the new behavior?
+
+## Checklist
+- [ ] My code follows the code style of this project.
```

Comment:
I'm not sure all the things in this checklist are applicable to this specific project.

Reply:
You're right! I removed the checklist as I felt it was unnecessary.